### DISCUSSION TITLE
Polish SIP translation

### DISCUSSION THREAD

**Discussion by kotjarka** [Dan-in-CA/SIP#302] [General]
Hi. First of all thanks for SIP, awesome software.
I prepared Polish translation files, based on .pot file. I hope you will include them in upcoming release.
  
BTW, during translating I found some untranslatable strings. Maybe you can fix that.
![obraz](https://user-images.githubusercontent.com/28064012/232853934-16689029-e187-4293-b996-e54ce0d905c6.png)

![obraz](https://user-images.githubusercontent.com/28064012/232862223-fb4457cd-3ddd-4e2e-8c0b-4b6ce7345918.png)

![obraz](https://user-images.githubusercontent.com/28064012/232862831-b92c7ba7-c4b2-417e-a9b3-a944f574f7b6.png)

![obraz](https://user-images.githubusercontent.com/28064012/232865372-cbef80bd-13e4-4d31-908e-12ac7b58bfe8.png)

![obraz](https://user-images.githubusercontent.com/28064012/232865848-b9bac3dd-01ee-4711-929c-20b67aca4692.png)

Also one of buttons (ODD/EVEN) in programs don`t change size to fit string
![obraz](https://user-images.githubusercontent.com/28064012/232855858-ceaa8c7d-72bd-4f1c-bc7e-50d851d633f3.png)
[pl_PL.zip](https://github.com/Dan-in-CA/SIP/files/11264978/pl_PL.zip)

**Comment by Dan-in-CA** [comment]
Greetings  kotjarka,

I will be more than happy to add your Polish translation to SIP.
A major new SIP update is nearly ready and I will see about fixing the problems you mention.

The new version of SIP will include a Node-RED extension that should make it much easier for users to add new features to SIP.
As a proof of concept I am ready to start testing a node-red based plugin that works with soil moisture sensors and implements a water conserving algorithm that has been proposed by researchers at the University of Warmia and Mazury in Olsztyn, Poland. So I am delighted to add a Polish translation.

Dan

**Comment by Dan-in-CA** [comment]
The Polish translation has been added to the P3-only branch. 
That branch includes the latest SIP updates. It will soon be merged into the main branch.

**Comment by Dan-in-CA** [comment]
kotjarka,
I made changes to SIP files that should take care of the text strings that did not translate.
There are also some strings from the new Node-red extension added.
attached is an update to the Polish PO file with the added strings. 
[sip_messages_update.zip](https://github.com/Dan-in-CA/SIP/files/11412975/sip_messages_update.zip)

**Comment by kotjarka** [comment]
Updated translation in attachment
[sip_messages_update.zip](https://github.com/Dan-in-CA/SIP/files/11415056/sip_messages_update.zip)

**Comment by Dan-in-CA** [comment]
Thank you kotjarka!
That was really quick.
There was a problem with the "Station" translation in Options > configure master.
It is fixed now in the P3-only branch.
Dan
